appmap/test/appmap_test_base.py

```diff
@@ -1,6 +1,64 @@
+import os
+import re
+
+import json
+
 import appmap._implementation
 
 
 class AppMapTestBase:
     def setup_method(self, _):
         appmap._implementation.initialize()  # pylint: disable=protected-access
+
+    def normalize_appmap(self, generated_appmap):
+        # Normalize paths
+        object_id = 1
+
+        def normalize(dct):
+            nonlocal object_id
+            if 'elapsed' in dct:
+                assert isinstance(dct['elapsed'], float)
+                dct['elapsed'] = 0.0
+            if 'git' in dct:
```

Comment:
Would it make sense to remove `git` instead, or set it to `None`? Perhaps after verifying the field types. (Same with `elapsed`.)

Reply:
@virajkanwade You added this filtering. What do you think about @dividedmind's suggestion?

Reply:
@apotterri @dividedmind Had initially thought of removing it, but then this atleast validates if the appropriate keys are being set.

Reply:
Done.